$OpenBSD$

Use something different than index as variable.
When using amalgation <string.h> is include and have already one index.
This supress a warning.

--- src/lib/eina_inlist.c.orig	Tue Oct 25 19:47:26 2011
+++ src/lib/eina_inlist.c	Tue Oct 25 19:50:36 2011
@@ -500,7 +500,7 @@ eina_inlist_sorted_state_free(Eina_Inlist_Sorted_State
 
 static void
 _eina_inlist_sorted_state_insert(Eina_Inlist_Sorted_State *state,
-                                 unsigned short index,
+                                 unsigned short idx,
                                  int offset)
 {
    Eina_Inlist *last;
@@ -509,10 +509,10 @@ _eina_inlist_sorted_state_insert(Eina_Inlist_Sorted_St
 
    state->inserted++;
 
-   if (offset != 0) index++;
-   for (; index < state->jump_limit; index++)
+   if (offset != 0) idx++;
+   for (; idx < state->jump_limit; idx++)
      {
-        state->jump_table[index] = state->jump_table[index]->prev;
+        state->jump_table[idx] = state->jump_table[idx]->prev;
      }
 
    start = state->jump_limit - 3;
@@ -542,7 +542,7 @@ _eina_inlist_sorted_state_insert(Eina_Inlist_Sorted_St
                }
 
              state->jump_table[start++] = last;
-             jump_count = 0;                  
+             jump_count = 0;
           }
      }
 }
